core/banking/transfers_common.go

```diff
@@ -77,7 +77,11 @@ func (e *Engine) CheckTransfer(t *types.TransferBase) error {
 	return nil
 }
 
-func (e *Engine) ensureMinimalTransferAmount(a *assets.Asset, amount *num.Uint) error {
+func (e *Engine) ensureMinimalTransferAmount(a *assets.Asset, amount *num.Uint, fromAccType types.AccountType) error {
+	if fromAccType == types.AccountTypeVestedRewards {
```

Comment:
I think the requirement is that only if it's the full amount then there's no minimum.

Reply:
I'm not sure that requirement makes all the sense, because you may want to take of some from the vested, but not all so you keep increase / conserve your multiplier.

Reply:
it just means that if the amount you want to transfer is so small (<min_amount), then we let you move it only if it's the full amount in the account. I don't know if the requirement makes sense or not, but it's what the ticket says :)

Reply:
hum right, I understood this as, all the balance, all the time not just if < min_amount. that makes more sense